.github/workflows/ci.yml

```diff
@@ -308,6 +308,39 @@ jobs:
     - name: Check `icrate` with each feature enabled separately
       run: cargo run --bin=check_icrate_features
 
+  check-icrate-examples:
+    name: Check icrate examples
+    runs-on: macos-12
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Use system Rust
+      run: cargo --version
+
+    - name: Cache Cargo
+      uses: actions/cache@v3
+      with:
+        path: ${{ env.CARGO_CACHE_PATH }}
+        key: cargo-${{ github.job }}-${{ matrix.name }}-${{ hashFiles('**/Cargo.lock') }}
+
+    - name: Check example `basic_usage`
+      run: cargo check --package=icrate --example=basic_usage --features=unstable-example-basic_usage
+
+    - name: Check example `browser`
+      run: cargo check --package=icrate --example=browser --features=unstable-example-browser
+
+    - name: Check example `delegate`
+      run: cargo check --package=icrate --example=delegate --features=unstable-example-delegate
+
+    - name: Check example `nspasteboard`
+      run: cargo check --package=icrate --example=nspasteboard --features=unstable-example-nspasteboard
+
+    - name: Check example `speech_synthesis`
+      run: cargo check --package=icrate --example=speech_synthesis --features=unstable-example-speech_synthesis
+
```

Comment:
I think adding the example feature to `unstable-frameworks-{whereever it works}` in `icrate/Cargo.toml` should be enough, don't need to change the CI here?

Reply:
> I think adding the example feature to `unstable-frameworks-{whereever it works}` in `icrate/Cargo.toml` should be enough, don't need to change the CI here?

Actually, maybe I'm misunderstanding but aren't the example features essentially covered already in that sense due to the `*_all` features? So for the WebKit example, `WebKit_all` is already included in `unstable-frameworks-*`.

In that case, wouldn't adding the example features be redundant?

Don't we also want to test that the examples actually build too?

Reply:
Well, every feature that the examples need is enabled, _except_ for the `unstable-example-...` feature. Cargo doesn't know that that feature is not actually required to compile the examples, so it doesn't try.

> Don't we also want to test that the examples actually build too?

I'm pretty sure that `cargo test` in `Test macOS 12` build all the examples as well (see e.g. [the latest build on `master`](https://github.com/madsmtm/objc2/actions/runs/4036432822/jobs/6939026601), search for `example`)

Reply:
Ah, okay, I see what you mean now.